pkg/lib/serverTime.js

```diff
@@ -258,42 +265,27 @@
         return status;
     };
 
-    const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
-                                                { superuser: "try" });
+    function get_timesync_backend() {
+        // If chronyd is installed, we assume that it takes precedence
+        // over timesyncd.
+        if (chronyd_service.exists)
+            return "chronyd";
+        else if (timesyncd_service.exists)
+            return "timesyncd";
+        else
+            return null;
+    }
+
+    function get_custom_ntp_timesyncd() {
+        const custom_ntp_config_file = cockpit.file("/etc/systemd/timesyncd.conf.d/50-cockpit.conf",
+                                                    { superuser: "try" });
 
-    self.get_custom_ntp = function () {
-        /* We only support editing the configuration of
-         * systemd-timesyncd, by dropping a file into
-         * /etc/systemd/timesyncd.conf.d.  We assume that timesyncd is
-         * used when:
-         *
-         * - systemd-timedated is answering for
-         *   org.freedesktop.timedate1 as opposed to, say, timedatex.
-         *
-         * - systemd-timesyncd is enabled (false if chrony is being used)
-         *
-         * The better alternative would be to have an API in
-         * o.fd.timedate1 for managing the list of NTP server
-         * candidates.
-         */
         const result = {
-            supported: false,
+            supported: true,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20230110-142000-984810ea-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)